closed/make/modules/ibm.healthcenter/Copy.gmk

```diff
@@ -32,7 +32,7 @@ $(HEALTHCENTER_EXTRACT) : $(HEALTHCENTER_JAR)
 	@$(RM) -rf $(@D)
 	$(call MakeDir, $(@D))
 	$(UNZIP) -q $< -d $(@D)
-  ifeq (zos,$(OPENJDK_TARGET_OS))
+  ifeq ($(call isTargetOs, zos), true)
```

Comment:
The copyright date needs to be updated in this file.

Reply:
It's been updated

Reply:
It has not; see line 2:
```
# (c) Copyright IBM Corp. 2021, 2021 All Rights Reserved
```